src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -3027,6 +3028,8 @@ PlainSelect PlainSelect() #PlainSelect:
             [ LOOKAHEAD(2) "ON" "(" distinctOn=SelectItemsList() { plainSelect.getDistinct().setOnSelectItems(distinctOn); } ")" ]
         )
         |
+        <K_DISTINCTROW> { Distinct distinct = new Distinct(); distinct.setUseDistinctRow(true); plainSelect.setDistinct(distinct); }
```

Comment:
This is separate from the `<K_DISTINCT>` case right above, because `DISTINCT ON` or `DISTINCTROW ON` is not applicable in MySQL.